.jenkins/pytorch/test.sh

```diff
@@ -283,4 +288,7 @@ else
   test_aten
   test_libtorch
   test_custom_script_ops
+  if [[ "${BUILD_ENVIRONMENT}" == *pynightly_test* ]]; then
+    test_python_ge_config_bailouts
```

Comment:
Why is this only enabled on nightly?

Reply:
I don't want to necessarily overburden our CI which is already under a lot of strain